src/pages/farmDeposit/farmDepositPage.tsx

```diff
@@ -147,7 +162,7 @@ const FarmDepositPage = (props: Props): ReactElement => {
           ></div>
           <InfoSection title="Stats" withDivider rows={statsDataRows} />
 
-          {poolType !== PoolTypes.LP && (<FarmInfoCard data={poolData} />)}
+          {poolType !== PoolTypes.LP && <FarmInfoCard data={poolData} />}
```

Comment:
lol it looks like somehow we have different prettier configs. Mine added the parenthesis I think.